package.json

```diff
@@ -70,6 +70,7 @@
     "@types/semver": "^7.3.8",
     "@typescript-eslint/eslint-plugin": "^4.28.3",
     "@typescript-eslint/parser": "^4.28.3",
+    "@typescript/lib-dom": "npm:@types/web",
```

Comment:
typescript 4.8 ships with an older dom lib, this gets us on the latest so that TS knows about the submitter

Reply:
Looks like none of the 5.0.x releases have it, but it's included in 5.1.3